packages/@o3r/rules-engine/builders/rules-engine-extractor/helpers/rules-engine.extractor.ts

```diff
@@ -45,26 +61,27 @@ export class RulesEngineExtractor {
   private readonly commentParser = new ConfigDocParser();
 
   constructor(tsconfigPath: string, private readonly basePath: string, private readonly logger: LoggerApi) {
-    const {config} = ts.readConfigFile(tsconfigPath, ts.sys.readFile);
+    // eslint-disable-next-line @typescript-eslint/unbound-method -- parameter of `readConfigFile` is of type `(path: string) => string | undefined`
+    const { config } = ts.readConfigFile(tsconfigPath, ts.sys.readFile);
```

Comment:
I assume the linter rule expects this instead
```suggestion
    const { config } = ts.readConfigFile(tsconfigPath, (p) => ts.sys.readFile(p));
```